src/main/java/by/exadel/internship/service/impl/InternshipServiceImpl.java

```diff
@@ -0,0 +1,55 @@
+package by.exadel.internship.service.impl;
+
+
+import by.exadel.internship.dto.internshipDTO.GuestInternshipDTO;
+import by.exadel.internship.entity.Internship;
+import by.exadel.internship.exception_handing.NotFoundException;
+import by.exadel.internship.mapper.InternshipMapper;
+import by.exadel.internship.repository.InternshipRepository;
+import by.exadel.internship.service.InternshipService;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.slf4j.MDC;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.UUID;
+
+@Service
+@RequiredArgsConstructor
+@Slf4j
+public class InternshipServiceImpl implements InternshipService {
+
+    private final InternshipRepository internshipRepository;
+    private final InternshipMapper internShipMapper;
+
+    @Override
+    public GuestInternshipDTO getById(UUID id) {
+
+        MDC.put("className", InternshipServiceImpl.class.getSimpleName());
+        log.info("Try to get Internships with id=" + id);
```

Comment:
I believe such service wrapping logging should be with debug level. Please review it everywhere

Reply:
Also discuss with team single strategy, because in some services I see such log method wrapping and in other it is missed. Stick to one style.

Reply:
> Also discuss with team single strategy, because in some services I see such log method wrapping and in other it is missed. Stick to one style.

yaaa, i added log to user service here: https://github.com/viktarkolbik/backend_team_four/pull/28, thanks.